res/layout/list_footer.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="56dp"
+    android:layout_height="112dp"
```

Comment:
It is done so that the List scrolls beyond the fab else it would hover above the last list item and for example that  fact that the last item is shared via link could not be seen in the list, see screenshot with the 1121dp height:
![device-2016-03-02-112556](https://cloud.githubusercontent.com/assets/1315170/13457661/90464b0c-e069-11e5-8b7b-0b5f1d5712c9.png)

With the height of 112 the fab is perfectly aligned in the footer with 16dp to the right, top and bottom :smiley: